packages/solid/src/validation/url.ts

```diff
@@ -12,9 +12,9 @@ type Value = MaybeValue<string>;
 export function url(error: string): (value: Value) => string {
   return (value: Value) => {
     try {
-      value && new URL(value);
+      new URL(value);
```

Comment:
This should be sufficient I think?

```ts
// All of these will throw an error and be caught in the catch
new URL(true);
new URL({});
new URL([]);
new URL(42);
new URL("0");
new URL("false");
new URL(new Date());
new URL(-42);
new URL(12n);
new URL(3.14);
new URL(-3.14);
new URL(Infinity);
new URL(-Infinity); 
```

Reply:
Values are optional by default with Modular Forms, so `value && ...` is required.

Reply:
`new URL(undefined)` should also throw as expected right?

Reply:
But we don't want to throw it in this case. Without `required()` there should be no error for missing or empty strings.

Reply:
Ah I see what you mean, thanks for the explanation